ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/macro/MacroRegistryImpl.java

```diff
@@ -39,9 +38,7 @@ public MacroRegistryImpl() {
     public void register(Set<Macro> macros) {
         for (Macro macro : macros) {
             final String name = macro.getName();
-            if (this.macros.containsKey(name)) {
-                Log.warn(MacroRegistryImpl.class, "Command macro '" + name + "' is already registered.");
```

Comment:
Ok. That's means that some problem has appeared recently.
So, thanks to this message, now we know that some code tries to register several macros with the same ID or the same macro registered several times. And we must react somehow to this problem (report an issue, fix in another PR).
But removing the log message doesn't solve the actual problem.